packages/v4/src/content/design-guidelines/components/skeleton/skeleton.md

```diff
@@ -6,7 +6,7 @@ section: components
 A **skeleton** is a type of loading state that allos you to expose content incrementally. For content that may take a long time to load, use a [progress bar](/components/progress/react) in place of a skeleton.
```

Comment:
Pre-existing typo I just caught where it says "allos" instead of "allows" - do you mind making that change?